pkg/storaged/client.js

```diff
@@ -1484,4 +1493,40 @@
     }
 };
 
+client.in_anaconda_mode = () => !!client.anaconda;
+
+client.strip_mount_point_prefix = (dir) => {
+    const mpp = client.anaconda?.mount_point_prefix;
+
+    if (dir && mpp) {
+        if (dir.indexOf(mpp) != 0)
+            return false;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19352-20231205-120952-f0d04c46-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)